### ISSUE TITLE
(Optional) PHP 7 support

### ISSUE THREAD

**Issue by EvanK** [evolution/wordpress#143]
Currently, we're using the stock PHP that Ubuntu 14 provides (~~5.6 IIRC~~ 5.5.9).

Look into adding a role for [PHP 7 installed from a ppa](https://www.digitalocean.com/community/tutorials/how-to-upgrade-to-php-7-on-ubuntu-14-04), that we can optionally swap out for the current php role.

**Comment by weeirishman** [comment]
The commands I ran on `local` to upgrade to PHP 7.1:

```
sudo add-apt-repository ppa:ondrej/php
sudo apt-get update
sudo apt-get install php7.1
sudo apt-get install php7.1-mysql php7.1-curl php7.1-gd php7.1-mcrypt php7.1-dev
sudo a2dismod php5
sudo a2enmod php7.1
sudo service apache2 restart
```

**Comment by EvanK** [comment]
@weeirishman Tagged & released as [v1.6.3](https://github.com/evolution/wordpress/releases/tag/v1.6.3)
